pkg/compactor/blocks_cleaner.go

```diff
@@ -493,3 +523,20 @@ func listBlocksOutsideRetentionPeriod(idx *bucketindex.Index, threshold time.Tim
 
 	return
 }
+
+// findMostRecentModifiedTimeForBlock finds the most recent modification time for all files in a block
+func findMostRecentModifiedTimeForBlock(ctx context.Context, blockID ulid.ULID, userBucket objstore.InstrumentedBucket, userLogger log.Logger) (time.Time, error) {
+	var modifiedTime time.Time
+	err := userBucket.Iter(ctx, blockID.String(), func(name string) error {
+		attrib, err := userBucket.Attributes(ctx, name)
+		if err != nil {
+			level.Error(userLogger).Log("msg", "couldn't get attribute for file in block", "block", blockID.String(), "file", name)
```

Comment:
Sorry for not being clear. I suggested to also remove this logging, since error will be logged in `cleanUserPartialBlocks` method too.